Update common-issues.md. Added common fixes for Searching for server issue. (#388)
SlimeVR/SlimeVR-Docs-Site: src/common-issues.md

If your SlimeVR GUI is repeatedly timing out from the SlimeVR server (check the logs), you may be able to fix this by running the following command in an administrator console: `netsh int tcp set supplemental internet congestionprovider=default`. This is caused by non-default Windows network configurations commonly used by modified OSes.

> ## Tracker shows "Searching for the server on the local network..." error repeatedly in the serial console / Tracker connects to wifi but not the server

Reviewer: ```suggestion
## Tracker shows "Searching for the server on the local network..." repeatedly in the serial console; connects to WiFi but not the server
```

Author: while this is gramatically correct, the other ones have the pattern i did 
![image](https://github.com/user-attachments/assets/f4dab1c8-7031-4585-a78a-09530ec82761)

Reviewer: Makes sense! Everything checks mewot meow.

Common fixes:
- [Make sure computer's Ethernet/WiFi connection is **set to Private**.](https://docs.slimevr.dev/common-issues.html#network-profile-is-currently-set-to-public)
- [Make sure **Network Discovery is enabled** on your active network interface.](https://www.asus.com/support/faq/1049382/)
- Disable any Antivirus or Internet Security software.
- Disable any VPN software or hardware.
- Make sure your trackers are not connected to a **Guest WiFi network**.
- Make sure the WiFi network does **NOT** have AP Isolation enabled.

> - Disable Windows Defender Firewall for Public and Private.

Reviewer: ```suggestion
- Disable Windows Defender Firewall for both Public and Private networks.
```

Uncommon fixes:

> - Deleting the SlimeVR configuration: Close SlimeVR and delete the configs folder at "%AppData%\dev.slimevr.SlimeVR"..

Reviewer: ```suggestion
- Deleting the SlimeVR configuration: Close SlimeVR and delete the configs folder at `%AppData%\dev.slimevr.SlimeVR`.
```

> - Installing the server then running the server on another device on the network, then closing the new server you installed causes the trackers to find the server old again.

Reviewer: ```suggestion
- Installing the server then running the server on another device on the network, then closing the new server you installed causes the trackers to reconnect to the previously used server.
```
